src/app/content/components/Page/highlightManager.ts

```diff
@@ -49,6 +56,9 @@ const onClickHighlight = (services: HighlightManagerServices, highlight: Highlig
   }
 
   services.getProp().focus(highlight.id);
+  if (services.getProp().scrollTarget) {
+    services.history.replace({hash: '', search: ''});
```

Comment:
we shouldn't be clearing the active search when they click on a highlight

if your goal here is to clear their selected search result, i don't feel like we have to go out of our way to do that until they set a new one, that is how browser behavior already works.